client/package.json

```diff
@@ -1,10 +1,10 @@
 {
 	"name": "ion-lsp-client",
 	"description": "VSCode LSP client for Ion. ",
```

Comment:
And make this `Ion Extension to support the Ion data format on VSCode`

Reply:
This description is for the client. I will change the description for the whole extension.